pkg/events/dispatcher.go

```diff
@@ -226,9 +226,9 @@ func (e EventHandler) Run(tickDuration time.Duration) {
 					cancelFuncs = append(cancelFuncs, cancelFunc)
 
 					if err := worker(ctx, event, eventLogger); err != nil {
-						eventLogger.log.WithError(err).Error("failed processing event")
 						if event.RetryCount > 5 {
 							eventLogger.log.Error("event reached max retries")
+							eventLogger.log.WithError(err).Error("failed processing event")
```

Comment:
```suggestion
						eventLogger.log.WithError(err).Info("failed processing event")
						if event.RetryCount > 5 {
							eventLogger.log.WithError(err).Error("failed processing event, reached max retries")
```
Trenger en info-linje for å ikke miste erroren i de forrige forsøkene. Kan hende det er forskjellige feil hver gang.